SafeguardSessions/SafeguardSessions.pq

```diff
@@ -156,7 +156,14 @@ HandleIfThereIsAnError = (input as record) =>
                 error input[Error]
         else
             let
-                logNotExpectedError = Logger.ErrorLog("An unexpected error happend during execution", input[Error])
```

Comment:
I would create a custom UnexpectedError wrapper class in the error module then call that like
`(try UnexpectedError(input[Error]))[Error]`

I think it works.

Reply:
Yeah, it works

`
let
    UnexpectedError = (dummy_param as any) => error Error.Record("a", "b", dummy_param),
    a = (try UnexpectedError([Dummy = "param"]))[Error]
in
    a
`

Reply:
ACK.

Reply:
I just noticed that the last paragraph of the commit message also needs to be updated.

Reply:
ACK.